apps/chat/channels.py

```diff
@@ -441,24 +419,18 @@ def _create_new_experiment_session(self):
         """Creates a new experiment session. If one already exists, the participant will be transfered to the new
         session
         """
-        if not self.experiment_session:
-            participant, _ = Participant.objects.get_or_create(identifier=self.chat_id, team=self.experiment.team)
-        else:
-            participant = self.experiment_session.participant
-        self.experiment_session = ExperimentSession.objects.create(
-            team=self.experiment.team,
-            participant=participant,
+        self.experiment_session = self.start_new_session(
             experiment=self.experiment,
-            llm=self.experiment.llm,
             experiment_channel=self.experiment_channel,
+            participant_identifier=self.participant_identifier,
+            session_status=SessionStatus.SETUP,
```

Comment:
I think it would make more sense to leave the `session_status` so it can default to `SessionStatus.ACTIVE`, since at this point in the conversation, we know that the participant sent a message, making the session active

Reply:
Changing this to `ACTIVE` breaks this test `apps.channels.tests.test_base_channel_behavior.test_pre_conversation_flow`. I'm going to leave it for now to preserve existing functionality.